components/kyma-operator/pkg/kymahelm/kyma-helm-client.go

```diff
@@ -182,9 +184,21 @@ func (hc *Client) UpgradeRelease(chartDir, releaseName, overrides string) (*rls.
 		helm.ReuseValues(false),
 		helm.UpgradeTimeout(3600),
 		helm.UpgradeWait(true),
+		helm.UpgradeCleanupOnFail(true),
 	)
 }
 
+//RollbackRelease performs rollback to given revision
+func (hc *Client) RollbackRelease(releaseName string, revision int32) (*rls.RollbackReleaseResponse, error) {
+	return hc.helm.RollbackRelease(
+		releaseName,
+		helm.RollbackWait(true),
+		helm.RollbackVersion(revision),
+		helm.RollbackCleanupOnFail(true),
+		helm.RollbackRecreate(true),
+		helm.RollbackTimeout(3600))
```

Comment:
imho `RollbackTimeout` and `UpgradeTimeout` should be global variables configurable by envs/flags

Reply:
was thinking about the same, will add TODO and implement it in follow-up